test/test_diffusion2D_multiphase.jl

```diff
@@ -34,9 +33,9 @@ using JustPIC, JustPIC._2D
 # Threads is the default backend,
 # to run on a CUDA GPU load CUDA.jl (i.e. "using CUDA") at the beginning of the script,
 # and to run on an AMD GPU load AMDGPU.jl (i.e. "using AMDGPU") at the beginning of the script.
-const backend = @static if ENV["JULIA_JUSTPIC_BACKEND"] === "AMDGPU"
+const backend = @static if ENV["JULIA_JUSTRELAX_BACKEND"] === "AMDGPU"
     JustPIC.AMDGPUBackend
-elseif ENV["JULIA_JUSTPIC_BACKEND"] === "CUDA"
+elseif ENV["JULIA_JUSTRELAX_BACKEND"] === "CUDA"
     CUDABackend
 else
     JustPIC.CPUBackend
```

Comment:
You still have a bunch of trailing `JustPIC.` calls which may need to be adapted to JustRelax.

Reply:
For AMDGPU and CPU it is needed as they are not exported by JP, however CUDA has its own

Reply:
Ah my bad maybe. So you also need to make selection for JustPIC in JustRelax wrt backend?

Reply:
JustPIC is not integrated within JustRelax -this may change in the future, but this keeps the pkg lighter as JustPIC is not strictly needed internally in JustRelax-, so both packages need their own backends to instantiate their `struct`s. However, it is enough to define only `ENV["JULIA_JUSTRELAX_BACKEND"]`, there is no need for `ENV["JULIA_JUSTPIC_BACKEND"]`.